lib/indico.js

```diff
@@ -30,9 +31,15 @@ indico.Collection = function(collectionName, config) {
 }
 
 indico.collections = function(config) {
-  return make_request('/custom/collections', {}, false, config);
+  return makeRequest('/custom/collections', {}, false, config);
 }
 
+indico.relevance = function(data, queries, config) {
```

Comment:
I'm not certain that this is the right way to go about it. Because all arguments are positional in node it's hard to abstract this out (I think we'd have to do something similar to defining another make_request method that takes in a different number of arguments).